src/main.js

```diff
@@ -104,8 +103,15 @@ function saveUserInput() {
 };
 
 function saveCurrentCover() {
-  if (!savedCovers.includes(currentCover))
+  if (!savedCovers.includes(currentCover)) {
   savedCovers.push(currentCover);
+  savedCoverSection.innerHTML +=
+  `<div class="mini-cover">
+    <img class="mini-cover" src=${currentCover.cover}>
+    <h2 class="cover-title">${currentCover.title}</h2>
+    <h3 class="tagline">A tale of <span class="tagline-1">${currentCover.tagline1}</span> and <span class="tagline-2">${currentCover.tagline2}</span></h3>
+  </div>`
+  }
```

Comment:
This function looks great! This is the event handler for the save cover button. 
It first checks if the currentCover is already in the savedCover array. If it is not already there, it pushes the currentCover to the array.
Then it will add all innerHTML elements to the savedCoverSection which is within the savedView.